Sources/UtilityBeltNetworking/Extensions/URLComponents+SetQueryItems.swift

```diff
@@ -36,11 +36,13 @@ extension URLComponents {
                 // Example: array[][][] (no indexes!)
                 self.evaluatedQueryItems(for: (key: "\(parameter.key)[]", value: $0))
             }
-        case let bool as Bool:
-            // TODO: Allow encoding of bools as numbers
-            return [URLQueryItem(name: parameter.key, value: "\(bool)")]
         case let value:
-            return [URLQueryItem(name: parameter.key, value: String(describing: value))]
+            // If the value is a boolean, display it as "true"/"false"
+            if let nsNumber = value as? NSNumber, CFGetTypeID(nsNumber as CFTypeRef) == CFBooleanGetTypeID() {
```

Comment:
Could the logic to see if an NSNumber is a boolean be extracted into an extension/computed property and unit tested?